### PR TITLE
feat: app responsive

### PR DESCRIPTION
**Pull request by PuneetP16**
Features:  Added mobile responsiveness
1. Mobile responsive aside bar.
2. Mobile responsive filter.

### CODE REVIEW

**Review by m26unkwn — APPROVED** [review]
Wonderful code written love the responsiveness of the website.

1. Moblie navigation is looking great.
2. Loading spinner is looking wonderful

Code is clean and readable and good to merge.

File: src/components/NoteLabel/NoteLabel.css [hunk 1]
```diff
@@ -3,9 +3,17 @@
 }
 
 .note_label__form {
+	/* position: absolute;
+	top: 6px;
+	right: 12.5rem; */
+
```
Comment (dhruvin25799) [inline src/components/NoteLabel/NoteLabel.css:9]: Can remove commented code here.

File: src/components/NoteLabel/NoteLabel.css [hunk 2]
```diff
@@ -3,9 +3,17 @@
...
 	position: absolute;
 	top: 6px;
-	right: 12.5rem;
+	right: 1.5rem;
+	border: 1px solid var(--COLOR-TXT);
+	border-radius: 8px;
+	z-index: var(--z-index-4);
+	outline: var(--COLOR-TXT);
 }
```
Comment (dhruvin25799) [inline src/components/NoteLabel/NoteLabel.css:17]: Liked the z-index stacking variables.
Good work.

File: src/contexts/noteContext.js [hunk 3]
```diff
@@ -139,7 +139,7 @@ export const NoteProvider = ({ children }) => {
 				note: note,
 			});
 		} catch (error) {
-			console.log("from addToNoteList", error);
+			console.log("from deleteArchivedNote", error);
 		}
```
Comment (dhruvin25799) [inline src/contexts/noteContext.js:143]: You can remove these console.log statements.

**Review by dhruvin25799 — APPROVED** [review]
Clean good, good UI.
Great work. Looks good to merge.
I've added some comments do check them out.

File: src/contexts/noteContext.js (hunk 3 above)
Comment (PuneetP16) [reply src/contexts/noteContext.js:143]: Ok

File: src/components/UI/Aside/Aside.css [hunk 4]
```diff
@@ -26,10 +26,45 @@
...
+	flex-flow: row nowrap;
+	grid-area: aside_mobile;
+	width: 100%;
+	align-items: center;
+}
+
+.nav--mobile .nav__items--mobile {
+	margin-left: 0;
+	display: flex;
+	flex-flow: row nowrap;
```
Comment (tarunsankhla) [inline src/components/UI/Aside/Aside.css:41]: Default is row no need to add

File: src/contexts/noteContext.js [hunk 5]
```diff
@@ -83,7 +83,7 @@ export const NoteProvider = ({ children }) => {
 
 			noteDispatch({ type: "DELETE", payload: output, note: note });
 		} catch (error) {
-			console.log("from addToNoteList", error);
+			console.log("from deleteNote", error);
```
Comment (tarunsankhla) [inline src/contexts/noteContext.js:86]: Remove Console and handle error

**Review by tarunsankhla — APPROVED** [review]

**Comment by PuneetP16** [comment]
Thanks for PR review

File: src/components/FilterPanel/FilterPanel.jsx [hunk 6]
```diff
@@ -2,6 +2,7 @@ import { useFilter, useNote } from "../../contexts";
 import { bxIcons } from "../../data/icons";
 import { Filteritem } from "../FilterItem/Filteritem";
 import "./FilterPanel.css";
+import "./FilterPanelMobile.css";
```
Comment (gauravsinhaweb) [inline src/components/FilterPanel/FilterPanel.jsx:5]: css file name should be smallcase!

File: src/components/UI/Aside/Aside.css [hunk 7]
```diff
@@ -26,10 +26,45 @@
...
+	justify-content: space-evenly;
+	align-items: center;
+	width: 100%;
+	color: var(--COLOR-TXT);
+}
+
+.nav--mobile .nav__list_item {
+	font-size: 3.2rem;
+}
+
```
Comment (gauravsinhaweb) [inline src/components/UI/Aside/Aside.css:52]: file is well structured!

File: src/pages/Landing/Landing.css [hunk 8]
```diff
@@ -38,3 +37,74 @@
...
+	body .hero__desktop {
+		display: none;
+	}
+
+	body .auth_btn {
+		padding: 8px 4rem;
+		font-size: 2rem;
+		margin-left: 0rem;
+	}
+
```
Comment (gauravsinhaweb) [inline src/pages/Landing/Landing.css:61]: not a good way for using html elem for specificity!

**Review by gauravsinhaweb — APPROVED** [review]
UI looks good,  some suggestions I mentioned, Good to merge after changes!

File: src/components/FilterPanel/FilterPanel.jsx (hunk 6 above)
Comment (PuneetP16) [reply src/components/FilterPanel/FilterPanel.jsx:5]: I am following same naming convention for jsx and css files.

File: src/pages/Landing/Landing.css (hunk 8 above)
Comment (PuneetP16) [reply src/pages/Landing/Landing.css:61]: Yeah that is to taken care of once i am done with all the functionality. Thanks for your review

**Pull request merged**